lib/application.js

```diff
@@ -284,7 +285,10 @@ app.handler = function(type, options) {
   var handler = this._handlers[type] = remotes.handler(type, options);
 
   remotes.classes().forEach(function(sharedClass) {
-    sharedClass.ctor.emit('mounted', app, sharedClass, remotes);
+    /* HACK!!! */
+    if (sharedClass.name !== 'ctrl') {
```

Comment:
Looks like it's used in the [loopback/lib/model.js file](https://github.com/strongloop/loopback/blob/363bc4d6c182067cf02fdf964ee4ea38ce408f35/lib/model.js#L810) to add the listener tree.

In theory, there's no reason to have a beforeRemote/afterRemote on a controller method right?